README.md

```diff
@@ -36,3 +36,28 @@ The command will write out a json file with the resulting DukeDS project id and
 ```
 python -m lando_util.upload <COMMAND_FILE> <OUTFILE>
 ```
+
+JSON command file:
+```
+{
+    "destination": "<project name>",
+    "readme_file_path": "<path to readme file>",
+    "paths": ["<Path to folder or file to upload>"],
+    "share": {
+        "dds_user_ids": ["<DukeDS user id to share with>"],
+        "auth_role": "<DukeDS auth role to grant to dds_user_ids users>",  // optional field
+        "user_message": "<Message to include in the share email>"  // optional field
+    },
+    "activity": {
```

Comment:
It looks like `activity` is a required field now, and that needs all of the nested fields (including a CWL JSON result file and resolved file version IDs). That seems a like a lot to require for uploading a list of files. Is it reasonable to break that into two steps? 

I think there's a lot of value in having a kubernetes-native DDS download and upload tool that isn't bespin-specific (or CWL workflow specific). Is it reasonable to break upload.py into steps (e.g. `upload`, `activity`, and `share`)? I know there are data from upload that need to be provided to activity and share, but maybe we could exchange that another way.

Reply:
Currently upload produces some DukeDS IDs consumed by activity/share that we would have to look up again (project id, uploaded file ids, etc) if moved into a separate step. I think breaking this up into `upload`, `activity` and `share` would be cleaner. Retry is much nicer when a job does only one thing.

Reply:
Sounds good. We can create an issue for  those enhancements and merge this then.